buildForDebugger.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# run this script script from the project root.  It will build with g++ -g compiler flags, so
+# gdb can be used to debug
+# Note: This is only tested on Ubuntu 20
+
```

Comment:
You probably want to configure this script to exit if one of the commands fails:
```suggestion
set -e
```

Reply:
OK, good point, I will add this